# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/thebes/moz.build
# Commit: c1bd0996764c
# Full Hash: c1bd0996764c49174c9169fe5550905ce5dcef88
# Author: Jed Davis <jld@mozilla.com>
# Date: 2021-07-06 21:42:42
# Regressor Bug: 1635451
# File Overlap Count: 1
# Description:
#   Bug 1635451 - Allow GLX to work in headless content processes. r=jgilbert
#   
#   This follows what we're already doing for EGL: a refcounted object
#   which can own the X connection, where we hold a weak reference from
#   the library object so that multiple contexts opportunistically share
# ==============================================================================

diff -r 2974b1ba2beb -r c1bd0996764c gfx/thebes/moz.build
--- a/gfx/thebes/moz.build	Tue Jul 06 07:42:41 2021 +0000
+++ b/gfx/thebes/moz.build	Tue Jul 06 07:42:42 2021 +0000
@@ -135,9 +135,13 @@
             "gfxXlibNativeRenderer.h",
             "gfxXlibSurface.h",
         ]
+        EXPORTS.mozilla.gfx += [
+            "XlibDisplay.h",
+        ]
         SOURCES += [
             "gfxXlibNativeRenderer.cpp",
             "gfxXlibSurface.cpp",
+            "XlibDisplay.cpp",
         ]
 
 elif CONFIG["MOZ_WIDGET_TOOLKIT"] == "windows":
